### PR TITLE
Start testing react components

### PR DESCRIPTION
**Pull request by ppikula**
I just wanted to start testing our react components. I found AirBnB library for this purpose: enzyme (the docs can be found here: http://airbnb.io/enzyme/docs/guides.html)

To be honest I don't know if these tests make much sense(I have impression that they are to much whitebox). I have zero experience in this field so all comments very welcomed.

### CODE REVIEW

File: priv/package.json [hunk 1]
```diff
@@ -36,13 +36,15 @@
     "bower": "1.8.0",
     "chai": "3.5.0",
     "css-loader": "0.23.0",
+    "enzyme": "^2.7.1",
```
Comment (afronski) [inline priv/package.json]: Remove `^`.

File: priv/test/call_tracer.test.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// eslint-env: mocha
+
+import React from "react";
+import ReactTestUtils from "react-addons-test-utils";
+import { shallow } from "enzyme";
+import { expect } from "chai";
+
+import { CallsTracer, CallsTableRow } from "../app/call_tracer.jsx";
+
+
```
Comment (afronski) [inline priv/test/call_tracer.test.jsx]: Superfluous empty line.

File: priv/test/call_tracer.test.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+function mockItem() {
+  return { call_time: 10, pid: "<0.1.0>", args: "[]", res: "result" };
+}
+
+// TODO before each
+describe("<CallsTableRow />", () => {
+  let el = null;
```
Comment (afronski) [inline priv/test/call_tracer.test.jsx]: Maybe `element` instead? ;)

File: priv/test/call_tracer.test.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  });
+
+  it("collapses after clicking button twice", () => {
+    let el2 = shallow(<CallsTableRow key="1" item={mockItem()}/>);
+
+    el2.find(".btn").simulate("click", { preventDefault() {} });
+    el2.find(".btn").simulate("click", { preventDefault() {} });
+
+    expect(el2.find(".glyphicon-chevron-right").exists()).to.equal(true);
+    expect(el2.prop("className")).to.equal("row-normal");
```
Comment (afronski) [inline priv/test/call_tracer.test.jsx]: I like those tests! Nice job.

What I find there a little bit cryptic and magic are those selectors e.g. (`.row-normal`). Maybe wrapping it into `const` with meaningful name, will increase readability?

Comment (ppikula) [inline priv/test/call_tracer.test.jsx]: True, will introduce constants.

File: priv/package.json (hunk 1 above)
Comment (ppikula) [inline priv/package.json]: Ok, I used `npm --save...` and got this.

**Pull request merged**